Sources/SwifterSwift/CoreGraphics/CGSizeExtensions.swift

```diff
@@ -82,9 +82,37 @@ public extension CGSize {
         return CGSize(width: lhs.width + rhs.width, height: lhs.height + rhs.height)
     }
 
-    /// SwifterSwift: Add a CGSize to self.
+    /// SwifterSwift: Add a tuple to CGSize.
+    ///
+    ///     let sizeA = CGSize(width: 5, height: 10)
+    ///     let result = sizeA + (5, 4)
+    ///     // result = CGSize(width: 10, height: 14)
+    ///
+    /// - Parameters:
+    ///   - lhs: CGSize to add to.
+    ///   - tuple: tuple value.
+    /// - Returns: The result comes from the addition of the given CGSize and tuple.
+    static func + (lhs: CGSize, tuple: (width: CGFloat, height: CGFloat)) -> CGSize {
+        return CGSize(width: lhs.width + tuple.width, height: lhs.height + tuple.height)
+    }
+
+    /// SwifterSwift: Add a tuple to CGSize.
     ///
     ///     let sizeA = CGSize(width: 5, height: 10)
+    ///     let result = (4, 5) + sizeA
+    ///     // result = CGSize(width: 9, height: 15)
+    ///
+    /// - Parameters:
+    ///   - scalar: tuple value.
+    ///   - rhs: CGSize to add to.
+    /// - Returns: The result comes from the addition of the given tuple and CGSize.
+    static func + (tuple: (width: CGFloat, height: CGFloat), rhs: CGSize) -> CGSize {
```

Comment:
As mentioned in the discussion, this version of the operator doesn't make sense. Imagine that with `+` you should be able to have an equivalent assignment operator (`+=`). In the assignment operator for a `CGSize` to the tuple, e.g.
```Swift
var size = (CGFloat(), CGFloat())
size += CGSize.zero
```
`size` would still be a tuple, not a `CGSize`. Therefore it doesn't make sense if the equivalent `+` operator returns a `CGSize`.
I recommend that you delete this method, don't just change the return type.

Reply:
Ok, I see. Will remove it.
Thanks.